services/authenticationService.js

```diff
@@ -57,10 +57,23 @@ class userService {
         localStorage.setItem('users', JSON.stringify(users));
     }
 
+    // checkIsUploadedPicture(email) {
+    //     let user = this.getUser(email);
+    //     console.log(user);
+    // }
+
+    uploadImage(email, imgId) {
+        let user = this.getUser(email);
```

Comment:
use const

Reply:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

read about const and how it works with objects and arrays